packages/i18n-calypso/src/test/index.js

```diff
@@ -307,6 +307,73 @@ describe( 'I18n', function () {
 		} );
 	} );
 
+	describe( 'formatCurrency', () => {
```

Comment:
The tests here should combined with the primitive ones above should have the same coverage as originally in format-currency package's. We basically test that the state handled properties (`locale` & `geoLocation`) propagate propagate properly into number-formatters.